docs/user-guide/api-mediation/api-mediation-onboard-an-existing-rest-api-service-without-code-changes.md

```diff
@@ -209,20 +215,52 @@ Following steps show how to add your service to the API Mediation Layer on your
 
         https://localhost:10010/application/routes
 
-    You should see a line:
+    The following line should appear:
 
         /api/v2/petstore/**: "petstore"
 
-    This line says that requests to relative gateway paths that start with `/api/v2/petstore/` are routed to the service with service ID `petstore`.
+    This line indicates that requests to relative gateway paths that start with `/api/v2/petstore/` are routed to the service with service ID `petstore`.
 
-    You successfully defined your Java application. If your service is running and you can access its endpoints. In case of the sample application it is for example:
+    You successfully defined your Java application if your service is running and you can access the service endpoints. The following example is the service endpoint for the sample application:
 
         https://localhost:10010/api/v2/petstore/pets/1
 
 
+## Add a definition in the API Mediation Layer in the Zowe runtime
+
+After you define and validate the service in YAML format, you are ready to add your service definition to the API Mediation Layer running as part of the Zowe runtime installation. 
+
+**Follow these steps:**
+
+1. Locate the Zowe runtime directory. The Zowe runtime directory was chosen during Zowe installation. 
+   Its location is in the `zowe-install.yaml` file in the variable `install:rootDir`. 
+
+    **Note:** We use the `${zoweRuntime}` symbol in following instructions.
+
+2. Copy your YAML file to the `${zoweRuntime}/api-mediation/api-defs` directory.
+
+3. Run your application. 
+
+4. Restart Zowe runtime or follow steps in section [(Optional) Reload the services definition after the update when the API Mediation Layer is already started](#optional-reload-the-services-definition-after-the-update-when-the-api-mediation-layer-is-already-started).
+
+5.  Go to the following URL to reach the API Gateway (default port 7554) and see the paths that are routed by the API Gateway:
+
+        https://${zoweHostname}:${gatewayHttpsPort}/application/routes
+
+   The following line should appear:
+
+        /api/v2/petstore/**: "petstore"
+
+    This line says that requests to the relative gateway paths that start with `/api/v2/petstore/` are routed to the service with service ID `petstore`.
+
+    You successfully defined your Java application if your service is running and you can access its endpoints. The endpoint displayed for the sample application is:
+
+        https://l${zoweHostname}:${gatewayHttpsPort}/api/v2/petstore/pets/1
+
+
 ## (Optional) Check the log of the API Mediation Layer
 
-The API Mediation Layer prints following messages to its log when the API definitions are processed:
```

Comment:
How do you check the log? "Issue the command ..."

Reply:
It is the log of the APIML that is printed when you start the APIML - e.g. in step 2 of section _Add and validate the definition in the API Mediation Layer running on your machine_. This log is printed to the console. I think that everyone following these steps will notice it. I can show it to you.